integ-test-promote.pipeline.yml

```diff
@@ -16,7 +16,7 @@ stages:
           matrix:
             microsoft_hosted_linux:
               poolName: Azure Pipelines
-              vmImage: ubuntu-latest
+              vmImage: ubuntu-22.04
```

Comment:
Should this change in `build-test-publish.pipeline.yml` too?

Reply:
I haven't seen too much trouble there because it doesn't run MATLAB, but it wouldn't hurt for consistency. I was slightly worried about some weird interactions with other Azure Tasks it calls if they haven't tested it on ubuntu-22 yet.